src/Processors/QtiV2/Out/Validation/LongtextValidationBuilder.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace LearnosityQti\Processors\QtiV2\Out\Validation;
+
+use LearnosityQti\Entities\QuestionTypes\longtext_validation;
+use qtism\common\enums\BaseType;
+use qtism\common\enums\Cardinality;
+use qtism\data\state\ResponseDeclaration;
+
+class LongtextValidationBuilder extends AbstractQuestionValidationBuilder
+{
+    
+    protected function buildResponseDeclaration($responseIdentifier, $validation)
+    {
+        /** @var longtext_validation $validation */
+        $responseDeclaration = new ResponseDeclaration($responseIdentifier);
+
+        $responseDeclaration->setCardinality(Cardinality::SINGLE);
+        $responseDeclaration->setBaseType(BaseType::STRING);
+
+        return $responseDeclaration;
+    }
+
+    public function buildValidation($responseIdentifier, $validation, $isCaseSensitive = true, $distractorRationaleResponseLevel = array())
+    {
+        $responseProcessing = null;
+        if (empty($validation) && (method_exists($validation, 'get_max_score')) && (empty($validation->get_max_score()))) {
```

Comment:
Can we remove redundant outer parentheses?
```
if (empty($validation) && method_exists($validation, 'get_max_score') && empty($validation->get_max_score())) {
```

Reply:
Done

Reply:
Hi @pooja31chaudhary ,
Have you pushed the fixes?